src/graphics/glcontroller.cpp

```diff
@@ -82,6 +88,17 @@ namespace graphics
 	void GLController::drawSimulation()
 	{
 		// TODO: render data from server
+		if (finalMesh)
+		{
+			cylinderSolidColorShader->use();
+			cylinderSolidColorShader->setMatrix("view", camera.getView());
+			cylinderSolidColorShader->setMatrix("projection", projection);
+			cylinderSolidColorShader->setMatrix("model", glm::mat4(1.0f));
+
+			glDisable(GL_CULL_FACE);
+			finalMesh->draw(cylinderSolidColorShader.get());
+			glEnable(GL_CULL_FACE);
+		}
```

Comment:
Dlaczego to jest w `drawSimulation`? Nie powinniśmy wyświetlać w tej funkcje streamu?

Reply:
To jest tymczasowe wyświetlanie finalnej wygenerowanej żyły do celów debugowania. Jak najbardziej masz rację, że tu będzie wyświetlany stream. Więc potem to wyrzucimy.

Reply:
Spoko